tests/unit/components/app-content-switcher/AppContentSwitcher.spec.jsx

```diff
@@ -0,0 +1,66 @@
+import { render, screen, fireEvent } from '@testing-library/react'
+import { describe, it, beforeEach } from 'vitest'
```

Comment:
```suggestion
```

Reply:
Unnecessary import was deleted.

Reply:
The 'expect' was added in the 2nd test, which checks if the function has been called.